.github/workflows/e2e.banyandb.yaml

```diff
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: BanyanDB
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 18 * * *'
+
+env:
+  SW_AGENT_JDK_VERSION: 8
+
+jobs:
+  StoragePlugin:
+    if: (github.event_name == 'schedule' && github.repository == 'apache/skywalking') || (github.event_name != 'schedule')
+    name: Storage Plugin
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Build Docker Image
+        shell: bash
+        env:
+          SKIP_TEST: "true"
+        run: |
+          echo "::group::Build Docker Image"
+          # Retry one more time due to frequent "maven connection reset"
+          (make docker || make docker) && (ES_VERSION=es7 TAG=latest-es7 make docker.oap || ES_VERSION=es7 TAG=latest-es7 make docker.oap)
+          echo "::endgroup::"
+      - name: Copy Dist Package
+        shell: bash
+        run: |
+          echo "::group::Copy Dist Package to test/e2e"
+          cp -R dist test/e2e/
+          echo "::endgroup::"
+      - name: Prepare E2E Test
+        shell: bash
+        run: |
+          echo "::group::Install SNAPSHOT apm-application-toolkit"
+          ./mvnw -DskipTests -N install
+          ./mvnw -f apm-application-toolkit -DskipTests -am install
+          echo "::endgroup::"
```

Comment:
You don't need this.

Reply:
I tried to run this locally. But I found that if I need to compile the e2e-provider, I have to install the `apm-application-toolkit` in the local repository.

Reply:
@kezhenxu94 Please give some suggestion. I think we need a guide line for new e2e-infra integration, which is already using in this branch.